tests/cpp/builder_tests.cpp

```diff
@@ -550,9 +550,13 @@ RuleWithCustomConditionWorks) {
 
 TEST_F(BuilderTests,
 RuleWithConditionWithSymbolsWorks) {
-	auto cond = forLoop(any(), "i", set({intVal(1), intVal(2), intVal(3)}), matchAt("$1", paren(entrypoint() + id("i")))).get();
+	auto cond = forLoop(any(), "n", set({intVal(1), intVal(2), intVal(3)}), matchAt("$1", paren(entrypoint() + id("i")))).get();
 	EXPECT_EQ("for", cond->getFirstTokenIt()->getPureText());
 	EXPECT_EQ(")", cond->getLastTokenIt()->getPureText());
+	auto forExp = std::static_pointer_cast<ForIntExpression>(cond);
+	EXPECT_EQ(forExp->getId(), "n");
+	forExp->setId("i");
```

Comment:
You can maybe add the check `EXPECT_EQ(forExp->getId(), "i");` after this line.

Reply:
Thank you for the review, I have added the line and will merge it after the checks are green.